web_src/js/index.js

```diff
@@ -2908,6 +2908,33 @@ function deSelect() {
 
 function selectRange($list, $select, $from) {
   $list.removeClass('active');
+
+  // add hashchange to permalink
+  const $issue = $('a.ref-in-new-issue');
+  const $copyPermalink = $('a.copy-line-permalink');
+
+  if ($issue.length === 0 || $copyPermalink.length === 0) {
+    return;
+  }
+
+  let updateIssueHref = function(anchor) {
+    const matched = $issue.attr('href').match(/%23L\d+$|%23L\d+-L\d+$/);
+    if (matched) {
+      $issue.attr('href', $issue.attr('href').replace($issue.attr('href').substr(matched.index), `%23${anchor}`));
+    } else {
+      $issue.attr('href', `${$issue.attr('href')}%23${anchor}`);
+    }
+  }
+
+  let updateCopyPermalinkHref = function(anchor) {
+    const matchedPermalink = $copyPermalink.attr('href').match(/#L\d+$|#L\d+-L\d+$/);
+      if (matchedPermalink) {
+        $copyPermalink.attr('href', $copyPermalink.attr('href').replace($copyPermalink.attr('href').substr(matchedPermalink.index), `#${anchor}`));
```

Comment:
I think the CI would fail, the indent doesn't match here?

I just thought some more simple code:

```
const href = $issue.attr('href');
href = href.replace(/%23L\d+$|%23L\d+-L\d+$/, '') + `%23${anchor}`;
$issue.attr('href', href')
```

The goal is to remove the trailing '#anchor' and add a new anchor.